### PR TITLE
[mapnik] update port

### PR DESCRIPTION
**Pull request by mathisloge**
- #### What does your PR fix?  
If mergeable this fixes some problems on windows and utf string encoding.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
yes

### CODE REVIEW

File: ports/mapnik/portfile.cmake [hunk 1]
```diff
@@ -4,7 +4,7 @@ vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO mapnik/mapnik
-    REF cf5541d180496628b8717306ef1f3fa87b89a886
+    REF 1d6350b1e5bcf684bc84d3451262581908261a0c
     SHA512 e82c3fd3da14ae37ebe04623ce03e8558326842aabd00fd8945a1db1dfff54b386a5e941996945981011d90cabbc3efbd2bffd49f55a34e1711940098629f70f
```
Comment (JonLiu1993) [inline ports/mapnik/portfile.cmake:8]: @mathisloge ,Thanks for your pr, looks you didn't update SHa512 ,Could you please update it?

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/mapnik/portfile.cmake (hunk 1 above)
Comment (mathisloge) [reply ports/mapnik/portfile.cmake:8]: @JonLiu1993 thanks for the review. will update after the PRs at upstream are merged

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the folloing triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**